pluggable-storage/src/main/java/pluggable/CustomStorageWrapper.java

```diff
@@ -28,7 +29,8 @@ public interface CustomStorageWrapper {
     void addItems(String key, List<String> items) throws Exception;
     void removeItems(String key, List<String> items) throws Exception;
     List<String> getItems(List<String> keys) throws Exception;
+    HashSet<String> getMembers(String key) throws Exception;
```

Comment:
I'd suggest using the `Set` interface instead, if possible
```suggestion
    Set<String> getMembers(String key) throws Exception;
```

Reply:
Updated.